docs/package.json

```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "@docusaurus/core": "2.4.1",
     "@docusaurus/preset-classic": "2.4.1",
-    "@stellar/freighter-api": "^1.5.0",
+    "@stellar/freighter-api": "^2.0.0",
```

Comment:
this will cause the build to break because this hasn't been released yet. But this will be resolved once the new version goes out